### DISCUSSION TITLE
UI Themes (Dark mode)

### DISCUSSION THREAD

**Discussion by braunjj** [dagster-io/dagster#18439] [Community]
We’re adding a dark mode (and an updated light mode) theme to the UI in the next release. These two new themes are experimental for now and you’ll be able to select a new theme from your user settings. 

In a future release we will remove the Legacy theme and the default theme will match your system settings. You’ll also be able to override this and always force dark mode or light mode.

We’re excited for you all to try the new themes. Let us know what you think! 

### Dark mode
<img width="1387" alt="image" src="https://github.com/dagster-io/dagster/assets/2798333/6bd5e3ca-b871-44e1-a634-07f1fdb7e31c">
<img width="1390" alt="image" src="https://github.com/dagster-io/dagster/assets/2798333/9102fe80-9d93-4312-ba79-53d0370f4cb2">

### Light mode
<img width="1388" alt="image" src="https://github.com/dagster-io/dagster/assets/2798333/05643eb5-a2dd-4265-8aa6-903dd33cfecd">
<img width="1392" alt="image" src="https://github.com/dagster-io/dagster/assets/2798333/e9fafd9b-c708-4661-9a61-179d40937f71">

### Theme Settings
<img width="1391" alt="image" src="https://github.com/dagster-io/dagster/assets/2798333/1fa11195-d426-44fa-881d-149a5e506d57">

**Comment by malow106** [comment]
It is so pleasing for my eyes !
